tools/pylib/_boutcore_build/boutcore.pyx.in

```diff
@@ -1719,12 +1769,19 @@ cdef class Datafile:
 
 
     def __dealloc__(self):
-        self.__mydealloc__()
+        self.__boutcore_dealloc()
 
-    def __mydealloc__(self):
+    def __boutcore_dealloc(self):
         # is never self owned
         pass
 
+def print(*args, sep=' ', end='\n'):
+    _print(sep.join([str(a) for a in args])+end)
+
+cdef _print(value):
+    cdef c.string str_=value.encode('utf-8')
+    c.output_info.write("{:s}", str_.c_str())
```

Comment:
```suggestion
    c.output_info.write(str_.c_str())
```
would probably work just as well? Or is this because the Python wrapper takes exactly two arguments?

Reply:
I wasn't sure - but I guess if it is resolved at compile time it should be safe?

With the printf like statement that would have resulted in potentially broken code ...